tests/functional/pages.py

```diff
@@ -554,6 +561,12 @@ def set_update_corrects_data_mistake(self, value):
         radio = element.find_element_by_xpath(f"//input[@value='{value}']")
         self.select_checkbox_or_radio(radio)
 
+    def set_corrected_measure_version(self, version):
+        element = self.driver.find_element(*EditMeasureLocators.UPDATE_CORRECTS_DATA_MISTAKE)
+        label = element.find_element_by_xpath(f"//label[contains(., '{version}')]")
+        radio = element.find_element_by_xpath(f"//input[@id='{label.get_attribute('for')}']")
+        self.select_checkbox_or_radio(radio)
```

Comment:
👏  this is great.

At some point, maybe we can refactor to a shared helper method along the lines of `select_radio_button_with_label(dom_node, text)`?